packages/types/src/transaction.ts

```diff
@@ -1,7 +1,9 @@
 import { DynamicSamplingContext } from './envelope';
 import { MeasurementUnit } from './measurement';
 import { ExtractedNodeRequestData, Primitive, WorkerLocation } from './misc';
+import { PolymorphicRequest } from './polymorphics';
```

Comment:
I looked at this type and something didn't feel right to me about `PolymorphicRequest` being an intersection instead of a union so I opened an issue to discuss it: https://github.com/getsentry/sentry-javascript/issues/5768

Of course that doesn't block this PR but just wanted to note it down here for posterity.